docs/prometheus-examples.md

```diff
@@ -0,0 +1,37 @@
+ - alerting.rules.yml
+ - alertmanager.service
+ - alertmanager.yml
+ - blackbox_exporter.service
+ - blackbox.yml
+ - prometheus.service
+ - prometheus.yml
+
+### Voting node health
+This `alerting.rules.yml` example has two alerts that can detect potentially transient failures. For example, our alerts will catch these problems:
+ - The node appears online, but `dcrd` has stalled (BlockHeightStalled)
+ - Block height is correct but `dcrwallet` doesn't agree with the other voting nodes (LiveTicketParity)
+
+On my testnet vsp, which uses extremely cheap virtual servers, the first alert frequently catches problems early.  The second scenario happens often, but usually resolves itself within a few minutes. So that alert is configured to wait for a much longer duration before firing.  When the duration is tuned appropriately, these alerts can inform you of a problem before `dcrstakepool` itself collapses.  At a minimum, when you find your `dcrstakepool` instance offline, you can pull up the Prometheus `/alerts` page and drill down to see which specific nodes aren't healthy.
+ 
+### Systemd service units
+These [service units](https://www.freedesktop.org/wiki/Software/systemd/) assume you've set up Prometheus under various `/opt` subdirectories, with binaries stored in `/opt/bin`.  They use an unprivileged user account to run the daemon. For your convenience, `ExecReload` has been configured so that you can reload configuration on the fly, e.g. 	`systemctl reload prometheus`.  Assuming you went through the quickstart already, here is how you could set up the `prometheus` daemon :
+```bash
+ sudo mkdir -p /opt/prometheus
+ sudo cp -r prometheus-2.13.1.linux-amd64/{consoles,console_libraries} /opt/prometheus/
+ sudo cp prometheus-2.13.1.linux-amd64/prometheus /opt/bin/
+ sudo useradd -Us /usr/bin/nologin -Md /opt/prometheus -c "Prometheus daemon" prometheus
+ sudo chown -R prometheus:prometheus /opt/prometheus
+ sudo cp prometheus.service /etc/systemd/system
+ sudo systemctl daemon-reload
+ sudo systemctl enable prometheus
+ sudo systemctl start prometheus
+```
+
+### Blackbox Exporter
+[Blackbox exporter](https://github.com/prometheus/blackbox_exporter/) is a Prometheus daemon you can deploy to monitor http endpoints like `dcrstakepool` itself. See the relevant job named "health-checks" in `prometheus.yml`.  The service unit assumes configuration is in a directory `/opt/blackbox_exporter` owned by the `prometheus` user like in the example above.
+
+### Mobile push notifications
+The provided `alertmanager.yml` example sends alerts to the api of a service called [pushover.net](https://pushover.net/). For one time fee of $5, you can use their api to send push notifications directly to your Android or iOS mobile device.  Alertmanager supports numerous notification methods, including a generic webhook api, PagerDuty (a significantly more expensive service), OpsGenie (free for up to 5 users), and more.
```

Comment:
Could you change "For a one time fee of $5" to something more ambiguous?  We can't promise that that fee will stay $5 or one time.  Maybe just "For a fee,"